src/main.rs

```diff
@@ -325,7 +362,7 @@ fn main() {
                     .collect::<Vec<_>>();
 
                 if chgss.iter().any(|chgs| chgs.iter().any(is_completion)) {
```

Comment:
I think with the recent addition of the `color!` macro/function, you could even remove this `if` and have something like:
```rust
println!(" → {}", color(colorize && chgss.iter().any(|chgs| chgs.iter().any(is_completion)), Blue, t.orig));
```
That said I'm not sure it looks better, so will let you judge on this :)
Maybe with adding an `is_completed_task(associated_tasks)` that does the `associated_tasks.iter().any(|c| c.iter().any(is_completion))` it'd look even better? Anyway, this could be left for later refactoring, so won't be a blocker :)